pkg/config/linters_settings.go

```diff
@@ -849,6 +849,7 @@ type TestifylintSettings struct {
 	Formatter            TestifylintFormatter            `mapstructure:"formatter"`
 	GoRequire            TestifylintGoRequire            `mapstructure:"go-require"`
 	RequireError         TestifylintRequireError         `mapstructure:"require-error"`
+	RequireStringMsg     bool                            `mapstructure:"require-string-msg"`
```

Comment:
@ldez, looks like dead (not used) attribute?
duplicates `TestifylintFormatter.RequireStringMsg`

Do I need open a separate PR for this? 😢